### PR TITLE
Homework 13.Kubernetes.Helm

### PR DESCRIPTION
**Pull request by Vigovskiyy**

### CODE REVIEW

File: Yuri_Vygovskiy/12.Kubernetes.First/runner/secret.yaml [hunk 1]
```diff
@@ -4,4 +4,8 @@ metadata:
   name: github-secret
 type: Opaque
 stringData:
+<<<<<<< HEAD
```
Comment (pluhin) [inline Yuri_Vygovskiy/12.Kubernetes.First/runner/secret.yaml:7]: Please clean here, looks like conflict's artefacts

**Review by pluhin — CHANGES_REQUESTED** [review]

File: Yuri_Vygovskiy/12.Kubernetes.First/runner/secret.yaml (hunk 1 above)
Comment (Vigovskiyy) [reply Yuri_Vygovskiy/12.Kubernetes.First/runner/secret.yaml:7]: done

**Comment by pluhin** [comment]
@Vigovskiyy 

You have deleted this file from PR but not from the history of your commits, this commit still contains your secret: [delete secret](https://github.com/pluhin/sa.it-academy.by/pull/1866/commits/6550f6b9f1d8c907950b918e4a8017fd7821ee6d)

it will be visible all time 

I explained theses solutions on our lessons:
- you can fixup (squash) your commit to remove this secret from the history
- revert  all commis from the head include this with secret, then add new one without this secrets

**Comment by Vigovskiyy** [comment]
I tried to make squash and fixup of problematic commits, but this greatly affected the entire commit history in my folder, after which I returned everything to its original state, but now git guardian sees 4 uncovered secrets. Could you help solve this problem, since I'm afraid to make changes already in order to avoid larger problems?

**Pull request merged**